apps/web/modules/booking/logs/views/booking-logs-view.tsx

```diff
@@ -51,9 +52,9 @@ type AuditLog = {
 interface BookingLogsFiltersProps {
     searchTerm: string;
     onSearchChange: (value: string) => void;
-    actorFilter: AuditActorType | null;
-    onActorFilterChange: (value: AuditActorType | null) => void;
-    actorOptions: Array<{ label: string; value: AuditActorType }>;
+    actorFilter: string | null;
```

Comment:
Actor Filter was earlier based on Actor Type which was wrong. It should be based on Actor Name to easily identify who made what changes.